src/Microsoft.OData.Core/UriParser/Aggregation/ApplyBinder.cs

```diff
@@ -98,9 +98,9 @@ private IEdmTypeReference CreateAggregateExpressionTypeReference(SingleValueNode
                 }
             }
 
-            switch (withVerb)
+            switch (withVerb.MethodType)

```

Comment:
Reverted the changes that affected signature of public methods. Please take a look to see if everything is OK..